### DISCUSSION TITLE
posts/use-sphinx-auto-generate-api-reference/

### DISCUSSION THREAD

**Discussion by giscus** [waynerv/waynerv.github.io#30] [Announcements]
# posts/use-sphinx-auto-generate-api-reference/

对于一个优秀的开源项目，高质量的文档与代码同样重要，糟糕的文档会将大部分潜在用户拒之门外。 从流行的开源项目中，我们可以总结出一些……

https://waynerv.com/posts/use-sphinx-auto-generate-api-reference/

**Comment by tanjuntao** [comment]
你好，请问下，博客中是为单独的 main.py 生成了 API 文档，如果是一个 Python package，package 下面有好多 sub-package，每个 sub-package 下面有好多 modules，这种情况下如何生成 API 文档，谢谢~
